packages/uint8arrays/src/lib/uint8arrays.spec.ts

```diff
@@ -1,4 +1,6 @@
+import { base64ToUint8Array } from 'packages/uint8arrays/dist/src/lib/uint8arrays';
 import { uint8arrayFromString, uint8arrayToString } from './uint8arrays';
+import { base64StringToBlob } from '../../../misc-browser/src/lib/misc-browser';
```

Comment:
This is triggering circular dependency error on build. 

1. change the import to `from '@lit-protocol-misc-browser`
2. In `project.json` add `"implicitDependencies": ["!misc-browser"],`